openssl-sys/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * Added `EVP_MD_CTX_size` and `EVP_MD_CTX_get_size`
 
+### Fixed
+
+* Fixed the deprecation note on `SSL_CTX_set_alpn_select_cb`.
```

Comment:
Do you prefer to write the CHANGELOG when cutting the release or in the PR?

Reply:
I scan through the commits to update the changelog when cutting a release but if people add it themselves in the PR that's fine too.